android/modules/android/src/java/ti/modules/titanium/android/AndroidModule.java

```diff
@@ -8,11 +8,16 @@
 
 import java.util.List;
 
+import android.Manifest;
```

Comment:
Unneeded imports due to 2nd PR moving it to a different namespace. Could you remove it?